security/keycloak-oidc-client-reactive-extended/src/main/resources/application.properties

```diff
@@ -16,6 +16,10 @@ quarkus.oidc-client.auth-server-url=${quarkus.oidc.auth-server-url}
 quarkus.oidc-client.client-id=test-application-client
 quarkus.oidc-client.credentials.secret=test-application-client-secret
 
+quarkus.oidc.authentication.user-info-required=true
+quarkus.oidc.authentication.scopes=openid,profile
+quarkus.oidc.roles.source=userinfo
```

Comment:
Where are roles check? if you map userinfo to roles (fine) you should use it or stop doing it, otherwise it seems like a random property set here.